src/openforms/registrations/contrib/objects_api/api.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any
+
+from django.urls import path
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import authentication, permissions, serializers, views
+
+from openforms.api.views import ListMixin
+
+from .client import get_objecttypes_client
+
+
+class ObjecttypeSerializer(serializers.Serializer):
+    # Keys are defined in camel case as this is what we get from the Objecttype API
```

Comment:
The JSON parser on the view does not convert it to snake case?

Reply:
We are using a custom `ListMixin` view in this case, which does everything "by hand": instantiating the serializer and returning a `Response` with the serializer data. In other words, the JSON parser/renderer is not involved :/

Reply:
we also don't want the renderer (not parser) to touch the output since we want to replicate the objecttypes API spec exactly for consistency reasons.